init-odkx-sync-endpoint.py

```diff
@@ -57,73 +61,73 @@ def run_interactive_config():
 
     return (enforce_https, env)
 
-def run_cache_setup(envParam) -> bool:
+def run_cache_setup(envParam: Dict[str, str]) -> bool:
 
-    env:Dict[str, str] = envParam
-    typer.echo("Please input the domain name you will use for this installation. A valid domain name is required for HTTPS without distributing custom certificates.")
-    input_domain: str = typer.prompt(f"domain [({env['HTTPS_DOMAIN']})]", default=env['HTTPS_DOMAIN'], show_default=False)
+    typer.echo(_("Please input the domain name you will use for this installation. A valid domain name is required for HTTPS without distributing custom certificates."))
+    input_domain: str = typer.prompt(_("domain [({envParam_https_domain})]").format(envParam_https_domain=envParam['HTTPS_DOMAIN']), default=envParam['HTTPS_DOMAIN'], show_default=False)
 
     if input_domain != "":
-        env['HTTPS_DOMAIN'] = input_domain
+        envParam['HTTPS_DOMAIN'] = input_domain
         save_progress('env', {'HTTPS_DOMAIN': input_domain})
 
 
     typer.echo("")
-    use_custom_password = typer.confirm("Do you want to use a custom LDAP administration password?")
+    use_custom_password = typer.confirm(_("Do you want to use a custom LDAP administration password?"))
     if use_custom_password:
         typer.echo("")
-        typer.echo("Please input the password to use for ldap admin")
-        default_ldap_pwd: str = typer.prompt("Ldap admin password", hide_input=True)
+        typer.echo(_("Please input the password to use for ldap admin"))
+        default_ldap_pwd: str = typer.prompt(_("Ldap admin password"), hide_input=True)
 
         if default_ldap_pwd != "":
             replaceInFile("ldap.env", r"^\s*LDAP_ADMIN_PASSWORD=.*$", "LDAP_ADMIN_PASSWORD={}".format(default_ldap_pwd))
-            typer.echo(f"Password set to: {default_ldap_pwd}")
+            typer.echo(_("Password set to: {default_ldap_pwd}").format(default_ldap_pwd=default_ldap_pwd))
+
+    typer.echo(_("Would you like to enforce HTTPS? We recommend yes."))
+    enforce_https = typer.confirm(_("enforce https?"), default=True)
 
-    typer.echo("Would you like to enforce HTTPS? We recommend yes.")
-    enforce_https = is_enforce_https()
```

Comment:
I think you want to keep is_enforce_https() a separate function to allow mocking it during testing, right?